code/script.js

```diff
@@ -2,59 +2,51 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
-
```

Comment:
Minor detail but, keeping your functions all together just for a easier read perhaps.

Reply:
I don't get it, they are all gathered at top with the rest of the getElemenById?

I think what you are referring to might be in some other section of the code?